### PR TITLE
vm-sysroot: provide a script for switching to production mode

### PR DESCRIPTION
**Pull request by sleirsgoevy**

### CODE REVIEW

**Comment by MikailBag** [comment]
Unfortunately PR is currently broken due to rust nightly compiler bug.
If you want this PR to be merged without blocking on #206 , you should cherry-pick changes from #206 that fix CI. AFAIK you need `scripts/ci-env.sh`

File: vm-sysroot/scripts/post-sysroot/jjs-prod.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+then if ! grep -q '^export JJS_ENV=prod$' /init
+    then
+        echo '/init has been modified, exiting'
+        exit 1
+    fi
+    echo 'Already running in production mode.'
+    exit 0
+fi
+echo 'Generating JJS_SECRET_KEY...'
+sed -i 's/^#export JJS_SECRET_KEY=$/export JJS_SECRET_KEY='"\$(dd if=/dev/urandom bs=64 count=1 | xxd -p -c 64)"'/g' /init
```
Comment (MikailBag) [inline vm-sysroot/scripts/post-sysroot/jjs-prod.sh]:
```suggestion
sed -i 's/^#export JJS_SECRET_KEY=$/export JJS_SECRET_KEY='"\$(dd if=/dev/random bs=64 count=1 | xxd -p -c 64)"'/g' /init
```

File: vm-sysroot/scripts/post-sysroot/jjs-prod.sh [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+echo 'Setting JJS_ENV=prod'
+sed -i 's/^export JJS_ENV=dev$/export JJS_ENV=prod/g' /init
+echo 'Done setting up. Reboot to apply changes.'
```
Comment (MikailBag) [inline vm-sysroot/scripts/post-sysroot/jjs-prod.sh]: Does host really have to be rebooted?

File: vm-sysroot/README.txt [hunk 3]
```diff
@@ -17,6 +17,9 @@ The resulting image is a single partition without any bootloader/kernel/whatsoev
 ./uml-build.sh
 Executes the two previous scripts, using UML to simulate root access. Doesn't require to be launched as root.
 
+./netns-build.sh
```
Comment (MikailBag) [inline vm-sysroot/README.txt]: The script is not defined in this patch

File: vm-sysroot/scripts/post-sysroot/jjs-prod.sh (hunk 2 above)
Comment (sleirsgoevy) [reply vm-sysroot/scripts/post-sysroot/jjs-prod.sh]: The daemons are started directly from the init script, so it's the most simple way to have them restarted.

File: vm-sysroot/README.txt (hunk 3 above)
Comment (sleirsgoevy) [reply vm-sysroot/README.txt]: I know, but I don't want to leave it undocumented.

File: vm-sysroot/scripts/post-sysroot/jjs-prod.sh (hunk 2 above)
Comment (MikailBag) [reply vm-sysroot/scripts/post-sysroot/jjs-prod.sh]: Mention is. Something like.
"Restart JJS components of reboot to apply config changed."

**Comment by MikailBag** [comment]
squash into 1 commit please

**Pull request merged**